docs/scenarios/stacks.feature

```diff
@@ -0,0 +1,138 @@
+Feature: As a library member of staff I want to login so I can place a request
+  # Note: users can currently login with either their Wellcome Trust account email address, library card number or library username
+
+  Scenario: User supplies correct user name and password
```

Comment:
The sign-in flow will be external to the site and be part of a provided service that we might consider unnecessary to specify here.

A change that might occur to the wellcomecollection.org site as a consequence of signing in _would_ be worth including though. e.g.

```feature
Given that I am a signed in user
When I visit the catalogue
Then I will see a link to my user profile
```

Encapsulating the "sign in" in a single step also makes it easier to test as we needn't mock interactions with an external service.